apps/api/src/app/events/services/storage-helper-service/storage-helper.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Injectable } from '@nestjs/common';
+import { IAttachmentOptions } from '@novu/stateless';
+import { StorageService } from '../../../shared/services/storage/storage.service';
+
+@Injectable()
+export class StorageHelperService {
+  constructor(private storageService: StorageService) {}
+
+  async uploadAttachments(attachments?: IAttachmentOptions[]) {
+    if (!(Array.isArray(attachments) && attachments.length > 0)) {
+      return;
+    }
```

Comment:
As this is being reused in all the methods of this class, we could take it to a private method to the class to be reused. Something in the lines of:
```
private areAttachmentsMissing(attachments?: IAttachmentOptions[]) {
  return !(Array.isArray(attachments) && attachments.length > 0);
}
```
so then in the methods we only need to:
```
 async uploadAttachments(attachments?: IAttachmentOptions[]) {
    if (this.areAttachmentsMissing(attachments)) {
      return;
    }
    ...
```

Reply:
Make Sense @p-fernandez, will update the same.